### DISCUSSION TITLE
Any ideas for Sekai Viewer?

### DISCUSSION THREAD

**Discussion by BentleyBentayga** [Sekai-World/sekai-viewer#410] [Ideas]
Sekai Viewer just updated with the korean server, i just really wanted to update the Sekai Viewer's version number.

The most important thing for this idea was the Sekai Wishlist, the wishlist has inactive in updates for TWO months, will @TitusEfferian update the Sekai Wishlist?

**Comment by kagami025** [comment]
the sekai ID cannot be binded，why？
